L1Trigger/L1TMuonOverlapPhase2/src/OmtfPhase2AngleConverter.cc

```diff
@@ -0,0 +1,31 @@
+#include "L1Trigger/L1TMuonOverlapPhase2/interface/OmtfPhase2AngleConverter.h"
+
+int OmtfPhase2AngleConverter::getProcessorPhi(int phiZero, l1t::tftype part, int dtScNum, int dtPhi) const {
+  int dtPhiBins = 65536;                    //65536. for [-0.5,0.5] radians
+  double hsPhiPitch = 2 * M_PI / nPhiBins;  // width of phi Pitch, related to halfStrip at CSC station 2
```

Comment:
I can't find if `nPhiBins` is known at compile time, but this seems like it could be another const or constexpr, as do several other values in this function. I won't be that picky about it though.

Reply:
No, `nPhiBins` is not known at compile time.